cmd/collector/app/span_processor.go

```diff
@@ -101,6 +101,7 @@ func (sp *spanProcessor) saveSpan(span *model.Span) {
 	if err := sp.spanWriter.WriteSpan(span); err != nil {
 		sp.logger.Error("Failed to save span", zap.Error(err))
 	} else {
+		sp.logger.Debug("Span written to the storage by the collector", zap.String("span-id", span.SpanID.String()))
```

Comment:
There is a way to pass Stringer to the logger without incurring the cost of conversion when debug is off.